src/Storage.cpp

```diff
@@ -1103,6 +1103,7 @@ Storage::Storage(const std::shared_ptr<const Options> & options_)
       subsmgr(new ScriptHashSubsMgr(options, this)),
       dspsubsmgr(new DSProofSubsMgr(options, this)),
       txsubsmgr(new TransactionSubsMgr(options, this)),
+      shtxsubsmgr(new ScriptHashTransactionsSubsMgr(options, this)),
```

Comment:
The manager has to live in the Sotrage scope, since Storage enqueues the notifications